test/unit/generator/content/part2/get-table/map-total-row.test.js

```diff
@@ -0,0 +1,125 @@
+const mapTotalRow = require('../../../../../../app/generator/content/part2/get-table/map-total-row')
+let totalRow
+let reductions
+
+describe('map total row', () => {
+  beforeEach(() => {
+    totalRow = JSON.parse(JSON.stringify(require('../../../../../mocks/statement-data').funding.find(x => x.name === 'Total')))
+    reductions = require('../../../../../mocks/reductions')
+  })
+
+  test('includes all table columns', () => {
+    const result = mapTotalRow(totalRow, reductions)
+    expect(result.length).toBe(8)
+  })
+
+  test('returns empty array if undefined total row', () => {
+    const result = mapTotalRow(undefined, reductions)
+    expect(result.length).toBe(0)
+  })
+
+  test('includes funding option', () => {
```

Comment:
going to mention here but overlaps with lots of test files

does funding option (in this case) not havva style associated? idk the pdfprinter but does it not use the default style in-place when 1 isn't given? should we add tests for tht if so?

Reply:
Done.  I've split the doc definition into it's own module and added some tests that cover those properties beyond the content.